<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(122, 82, 208);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>560</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="labelTelaLogin">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Verdana&quot;;
</string>
       </property>
       <property name="text">
        <string>Faça seu login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputCPFLogin">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>180</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font: 8pt &quot;Verdana&quot; ;
	border: 2px solid rgb(130, 130,130);
	border-radius: 20px;
	padding: 15px;
	background-color: rgb(255, 255, 255);
	color: rgb(122, 82, 208);
}

QLineEdit:hover{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(122, 82, 208);
}

QLineEdit:focus{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(122, 82, 208);
}</string>
       </property>
       <property name="maxLength">
        <number>11</number>
       </property>
       <property name="placeholderText">
        <string>CPF</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputSenhaLogin">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font: 8pt &quot;Verdana&quot; ;
	border: 2px solid rgb(130, 130,130);
	border-radius: 20px;
	padding: 15px;
	background-color: rgb(255, 255, 255);
	color: rgb(122, 82, 208);
}

QLineEdit:hover{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(122, 82, 208);
}

QLineEdit:focus{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(122, 82, 208);
}</string>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>SENHA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonLogin">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>340</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 8pt &quot;Verdana&quot; ;
	border: 2px solid rgb(122, 82, 208);
	border-radius: 5px;
	padding: 15px;
	color: rgb(122, 82, 208);
}

QPushButton:pressed{
	background-color: rgb(122, 82, 208);
	color: rgb(0, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>CONTINUAR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonEsqSenha">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>430</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(122, 82, 208);
}

QPushButton:hover{
	color: rgb(91, 222, 229);
}</string>
       </property>
       <property name="text">
        <string>Esqueci minha senha &gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonNoClient">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>470</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(122, 82, 208);
}

QPushButton:hover{
	color: rgb(91, 222, 229);
}</string>
       </property>
       <property name="text">
        <string>Ainda não sou cliente &gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelLoginStatus">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>150</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 200, 200);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
